.ci/pipelines/build.Jenkinsfile

```diff
@@ -41,9 +41,13 @@ pipeline {
                     }
                 }
                 stage('Upload YAML manifest to S3') {
+                    environment {
+                        VERSION="${sh(returnStdout: true, script: '. ./.env; echo $IMG_VERSION').trim()}"
+                    }
+
                     steps {
                         script {
-                            env.VERSION = readFromEnvFile("VERSION")
+                            sh 'echo VERSION=$VERSION'
```

Comment:
Nit: what's the purpose of this echo?
```suggestion
```

We can already see the version in different places:
```sh
[2021-03-16T10:14:41.784Z] [Pipeline] }
[2021-03-16T10:14:41.800Z] [Pipeline] // stage
[2021-03-16T10:14:41.816Z] [Pipeline] stage
[2021-03-16T10:14:41.820Z] [Pipeline] { (Upload YAML manifest to S3)
[2021-03-16T10:14:41.846Z] [Pipeline] sh
[2021-03-16T10:14:42.126Z] + . ./.env
[2021-03-16T10:14:42.126Z] + REGISTRY_NAMESPACE=eck-snapshots
[2021-03-16T10:14:42.126Z] + IMG_SUFFIX=
[2021-03-16T10:14:42.126Z] + GO_TAGS=release
[2021-03-16T10:14:42.126Z] + export LICENSE_PUBKEY=/go/src/github.com/elastic/cloud-on-k8s/.ci/license.key
[2021-03-16T10:14:42.126Z] + SNAPSHOT=true
[2021-03-16T10:14:42.126Z] + IMG_VERSION=1.6.0-SNAPSHOT-2021-03-16-a1026cfa
[2021-03-16T10:14:42.126Z] + echo 1.6.0-SNAPSHOT-2021-03-16-a1026cfa
[2021-03-16T10:14:42.147Z] [Pipeline] withEnv
[2021-03-16T10:14:42.151Z] [Pipeline] {
[2021-03-16T10:14:42.170Z] [Pipeline] script
[2021-03-16T10:14:42.176Z] [Pipeline] {
[2021-03-16T10:14:42.192Z] [Pipeline] sh
[2021-03-16T10:14:42.473Z] + echo VERSION=1.6.0-SNAPSHOT-2021-03-16-a1026cfa
[2021-03-16T10:14:42.473Z] VERSION=1.6.0-SNAPSHOT-2021-03-16-a1026cfa
[2021-03-16T10:14:42.487Z] [Pipeline] sh
[2021-03-16T10:14:42.768Z] + make -C .ci yaml-upload
[2021-03-16T10:14:42.768Z] make: Entering directory '/var/lib/jenkins/workspace/cloud-on-k8s-pr-full/.ci'
[2021-03-16T10:14:43.336Z] make[1]: Entering directory '/var/lib/jenkins/workspace/cloud-on-k8s-pr-full/.ci'
[2021-03-16T10:14:44.274Z] docker.elastic.co/eck-ci/eck-ci-tools:3a8abc46
[2021-03-16T10:14:47.565Z] Completed 219.4 KiB/219.4 KiB (554.6 KiB/s) with 1 file(s) remaining
upload: config/all-in-one.yaml to s3://download.elasticsearch.org/downloads/eck/1.6.0-SNAPSHOT-2021-03-16-a1026cfa/all-in-one.yaml
[2021-03-16T10:14:47.565Z] make[1]: Leaving directory '/var/lib/jenkins/workspace/cloud-on-k8s-pr-full/.ci'
[2021-03-16T10:14:47.565Z] make: Leaving directory '/var/lib/jenkins/workspace/cloud-on-k8s-pr-full/.ci'
```

Reply:
It's only to preserve the behaviour of `readFromEnvFile`: https://github.com/elastic/cloud-on-k8s/blob/dbba0de9d3ce771678d09db5be299da56afd796d/.ci/pipelines/build.Jenkinsfile#L156

I suppose it is useful to see which version is being uploaded without having to scroll back and forth. Happy to remove it if you think it's not necessary.

Reply:
Huhu, I'm the author of the initial `echo`. I would remove it.